### ISSUE TITLE
Resolve build warnings

### ISSUE THREAD

**Issue by peterdesmet** [gbif/doc-camera-trap-guide#25]
The [build](https://builds.gbif.org/job/doc-camera-trap-guide/lastBuild/console) currently returns the following potential issues

- [ ] asciidoctor: INFO: possible invalid reference: deployment
- [ ] asciidoctor: INFO: possible invalid reference: sequence
- [ ] asciidoctor: INFO: possible invalid reference: media
- [x] asciidoctor: WARNING: problem encountered in image: /documents/img/doc-camtrap-illustration.svg; Error retrieving URL ../../../../../../../../../../../../Users/javiergamboa/Library/CloudStorage/Box-Box/Personal-jagm/2022/Design/Collage/Archivo/Cutouts/Textures/textRecurso 1.png: No handler available for this URL scheme
- [x] Unknown reference: waller-2020
- [x] asciidoctor: WARNING: 400.en.adoc: line 288: id assigned to block already in use: table-occurrence-core
- [x] asciidoctor: WARNING: 400.en.adoc: line 724: id assigned to block already in use: table-ac-extension
- [x] asciidoctor: WARNING: <stdin>: line 19: id assigned to anchor already in use: camtrap-dp
- [x] asciidoctor: INFO: possible invalid reference: section-occurrenceRemarks

**Comment by peterdesmet** [comment]
@MattBlissett any idea how we could resolve the above mentioned INFO / WARNINGs in the build?

**Comment by MattBlissett** [comment]
The "INFO" ones seem to be caused by this, so I wouldn't worry about them: https://github.com/asciidoctor/asciidoctor/issues/3597

I've fixed the SVG warning. The SVG has a bitmap image included by reference to a location on Javier's computer. Other than the circle it's all bitmaps anyway, so I've just used the PNG for the PDF cover.

You might want to change the :revnumber: and :revdate: attributes, so the version shown in the document isn't just a Git revision. Or you might like it that way.
